test/models/test_cice5.py

```diff
@@ -184,15 +184,40 @@ def test_setup_fails(config, cice_config_files):
         with pytest.raises(Exception):
             model.setup()
 
+# with no restart_dir, payu should fail as payu just move the restart_dir wholesale for setup/archve
+NORES_CICE_NML = deepcopy(DEFAULT_CICE_NML)
+del NORES_CICE_NML["setup_nml"]["restart_dir"]
+
+@pytest.mark.parametrize("config", 
+                        [DEFAULT_CONFIG],
+                         indirect=True)
+@pytest.mark.parametrize("cice_config_files", 
+                        [NORES_CICE_NML],
+                         indirect=True)
+def test_Experiment_fails(config, cice_config_files):
+    """
+    Confirm that payu fails with a missing restart dir
+    """
+    with cd(ctrldir):
+
+        with pytest.raises(Exception):
+            lab = payu.laboratory.Laboratory(lab_path=str(labdir))
+            expt = payu.experiment.Experiment(lab, reproduce=False)
+
```

Comment:
```suggestion
        lab = payu.laboratory.Laboratory(lab_path=str(labdir))
        with pytest.raises(RuntimeError, match='`restart_dir`'):
            expt = payu.experiment.Experiment(lab, reproduce=False)
```

Non crucial suggestion just to make sure other exceptions don't cause the test to pass